### PR TITLE
Add rules to configure zIPL

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Add `bootloader-zip` group
- Add few rules for kernel options in zIPL
  - Ensure `audit=1` is set
  - Ensure `audit_backlog_limit=8192` is set
  - Ensure `selinux=0` is not set

### CODE REVIEW

File: linux_os/guide/system/bootloader-zipl/zipl_audit_argument/rule.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    this for all processes which launch after it does, adding the kernel argument
+    ensures it is set for every process during boot.
+
+severity: medium
+
+ocil_clause: 'auditing is not enabled at boot time'
+
+ocil: |-
+  To check that all boot entries enable audit at boot time, run the following command:
+  <pre>sudo grep -L "\baudit=1\b" /boot/loader/entries/*.conf</pre>
```
Comment (dahaic) [inline linux_os/guide/system/bootloader-zipl/zipl_audit_argument/rule.yml:30]: Does it makes sense to have ^options.* there as well? To be really sure we are checking the right line.

File: linux_os/guide/system/bootloader-zipl/zipl_enable_selinux/rule.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    Disabling a major host protection feature, such as SELinux, at boot time prevents
+    it from confining system services at boot time.  Further, it increases
+    the chances that it will remain off during system operation.
+
+severity: medium
+
+ocil_clause: 'SELinux is disabled at boot time'
+
+ocil: |-
+    Inspect <tt>/etc/default/grub</tt> for any instances of <tt>selinux=0</tt>
```
Comment (ggbecker) [inline linux_os/guide/system/bootloader-zipl/zipl_enable_selinux/rule.yml:25]: Does it make sense to have this `grub` thing?

Comment (yuumasato) [reply linux_os/guide/system/bootloader-zipl/zipl_enable_selinux/rule.yml:25]: No, :joy:

File: linux_os/guide/system/bootloader-zipl/zipl_vsyscall_argument/rule.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    To ensure that new kernels and boot entries continue to disable virtual syscalls,
+    add <tt>vsyscall=none</tt> to <tt>/etc/kernel/cmdline</tt>.
+
+rationale: |-
+    Poisoning writes an arbitrary value to freed pages, so any modification or
+    reference to that page after being freed or before being initialized will be
+    detected and prevented.
+    This prevents many types of use-after-free vulnerabilities at little performance cost.
+    Also prevents leak of data and detection of corrupted memory.
```
Comment (jan-cerny) [inline linux_os/guide/system/bootloader-zipl/zipl_vsyscall_argument/rule.yml:23]: Please change this to a rationale for vsyscalls.

Comment (yuumasato) [reply linux_os/guide/system/bootloader-zipl/zipl_vsyscall_argument/rule.yml]: Nice catch, fixed.

**Pull request merged**